satisfactory_calculator.py

```diff
@@ -28,64 +32,79 @@ def __str__(self):
         if len(self.components) == 0:
             return self.name
         per_base_unit = ', '.join(f'{k}: {v:g}' for k,v in self.base_units.items())
+        per_min = ', '.join(f'{v*self.per_min_mult:g} {k}/min' for k,v in self.base_units.items())
         output = f'{self.name}'
         output += f'  ({per_base_unit})'
+        output += f'  [{per_min}]'
         return output
 
     def __repr__(self):
         return f'<{self.__class__.__name__}: {str(self)}>'
 
-iron_ore = Thing('iron')
-iron_ingot = Thing('iron-ingot', {iron_ore: 1})
-iron_plate = Thing('iron-plate', {iron_ingot: 1.5})
-iron_rod = Thing('iron-rod', {iron_ingot: 1})
-screw = Thing('screw', {iron_rod: .25})
-reinforced_plate = Thing('reinforced-plate', {iron_plate: 6, screw: 12})
-rotor = Thing('rotor', {iron_rod: 5, screw: 25})
-smart_plating = Thing('smart-plating', {reinforced_plate: 1, rotor: 1})
-modular_frame = Thing('modular-frame', {reinforced_plate: 1.5, iron_rod: 6})
+class ThingPerMin(ThingRecipe):
+    def __init__(self, name, amount_produced_per_min=None, components_per_min={}):
```

Comment:
This would probably benefit from some comments or more expressive variable names, it took me a little while discern what was being done.

Also `1 / amount_produced_per_min * 60` is equivalent to `60 / amount_produced_per_min`. I would prefer the simplified version while you're at it.

And because `time_to_produce = 60 / amount_produced_per_min` it follows that `v / 60 * time_to_produce = v / amount_produced_per_min`. I feel like that expression better describes what we're really doing here, trying to arrive at the sub components per component, given the sub components per minute and the components per minute.